packages/create-astro/src/messages.ts

```diff
@@ -121,6 +121,9 @@ export const nextSteps = async ({ projectDir, devCmd }: { projectDir: string; de
 
 	await sleep(100);
 	if (projectDir !== '') {
+		if (projectDir.includes(' ')) {
+			projectDir = `"${projectDir}"`;
+		}
 		const enter = [
 			`\n${prefix}Enter your project directory using`,
 			color.cyan(`cd ./${projectDir}`, ''),
```

Comment:
This should use a real shell-quoting function like ``quote(["cd", `./${projectDir}`])`` from the [shell-quote](https://www.npmjs.com/package/shell-quote) library, rather than a one-off hack for just one of many shell metacharacters that might be misinterpreted.